hacker_news/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class New(models.Model):
```

Comment:
Oh that...I'll fix that...It can be done even after keeping the name News

On Sat, 7 May 2016 20:36 Arunabh, <notifications@github.com> wrote:

> In hacker_news/models.py
> <https://github.com/stabiitb/oksp/pull/5#discussion_r62417495>:
>
> > @@ -0,0 +1,25 @@
> > +from django.db import models
> > +
> > +
> > +class New(models.Model):
>
> Thats just for consistency with Django admin. Otherwise it shows Newss in
> Django admin
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/stabiitb/oksp/pull/5/files/2793e8951c0fe7f8f1b5cb1217915be5fbc6a0b9#r62417495>
>
-- 
First Year Undergraduate,
Computer Science,
IIT Bombay.